packages/lz4.rb

```diff
@@ -3,22 +3,22 @@
 class Lz4 < Package
   description 'LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core (0.16 Bytes/cycle).'
   homepage 'https://lz4.org/'
-  version '1.9.4'
+  version '1.10.0'
   license 'BSD-2 and GPL-2'
   compatibility 'all'
-  source_url 'https://github.com/lz4/lz4/archive/v1.9.4.tar.gz'
-  source_sha256 '0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b'
-  binary_compression 'tar.zst'
+  source_url "https://github.com/lz4/lz4.git"
+  git_hashtag "v#{version}"
+  binary_compression 'tar.xz'
 
   binary_sha256({
-    aarch64: '8248d98221d940f8a06a0c19562572b5128afe214b9eb83e4c0fc5240706c2e1',
-     armv7l: '8248d98221d940f8a06a0c19562572b5128afe214b9eb83e4c0fc5240706c2e1',
-       i686: 'ab796d7ebec72971369520ff7a7a87cbdf1c76cd04a2a095c2bf712e72741c5a',
-     x86_64: '76a2f4eccf84e78f32bbd2f8e6042f3ed4cda506ca74706fbbc4d60488b54a2a'
+    aarch64: '441620cf23567ee829c2c61e44c96f3230aa5df9780ef46708a16da3c1d13984',
+     armv7l: '441620cf23567ee829c2c61e44c96f3230aa5df9780ef46708a16da3c1d13984',
+       i686: 'c1ee1b29762fe352bb1fa588b5ae8ddc4589b45479bebe0723449258fabac7b2',
+     x86_64: '6e07a8d095f778015b2e30681f3d17e37bbd33b13c52f1ae2468e3f90a85cffd'
   })
 
+  no_zstd
```

Comment:
Can we put the boolean properties below the depends_on lines to be consistent with the other packages?  It gets confusing, otherwise.

Reply:
Done.